### ISSUE TITLE
FixedSizeList demo shows list is limited by floating point precision

### ISSUE THREAD

**Issue by glromeo** [bvaughn/react-window#182]
To reproduce this just go in the codepen and use:
```
const Example = () => (
  <List
    className="List"
    height={75}
    itemCount={100000}
    itemSize={600}
    layout="horizontal"
    width={1000}
  >
    {Column}
  </List>
);
```
Chrome 72 64bit Windows 10:

![image](https://user-images.githubusercontent.com/160981/54475801-b02bea00-47ed-11e9-9d48-4e063849bd98.png)

**Comment by glromeo** [comment]
Chrome 72 64bit on Mac OSX:

![image](https://user-images.githubusercontent.com/160981/54475832-157fdb00-47ee-11e9-853b-1da4fa8e0227.png)

**Comment by bvaughn** [comment]
This is a browser limitation, not a react-window bug. The same behavior can be seen with pure HTML: https://codesandbox.io/s/j2xkl9rkx9

In past libraries (react-virtualized) I've tried to accommodate these limitations by [scaling scroll offsets](https://github.com/bvaughn/react-virtualized/blob/master/source/Grid/utils/ScalingCellSizeAndPositionManager.js) but I decided against an approach like that for this library.

It is my belief that 60M pixel wide (or tall) lists aren't very realistic and wouldn't be very user friendly anyway. Some additional form of filtering is needed to make that amount of data browsable in any meaningful way.

Closing as "won't fix".
